external/dhcpd/dhcpd.c

```diff
@@ -321,7 +321,7 @@ static char DHCPD_IFNAME[IFNAMSIZ] = { 0, };
 static struct timeval g_select_timeout = { 1, 0 };
 #endif
 
-static dhcp_sta_joined g_dhcp_sta_joined;
+static dhcp_sta_joined g_dhcp_sta_joined = NULL;
 
```

Comment:
Can keep this uninitialized and move initialize code 
g_dhcp_sta_joined = NULL;  to dhcpd_start function

Reply:
Could you tell me what benefit can I get from that?

Reply:
In current situation, if dhcpd_stop is called the above variable remains initialized to the callback even after stop. Next if  start is called with NULL then the above pointer still remains initialized to old call back. So if we move it to start function it would be good. Handling is done in one place, unlike  earlier in 2 places.